src/features/content-explorer/content-explorer/ContentExplorerActionButtons.js

```diff
@@ -74,6 +77,33 @@ const ContentExplorerActionButtons = ({
         }
     };
 
+    const getStatusElement = statusMessage => {
+        let statusElement = <span className="status-message">{statusMessage}</span>;
+
+        if (onViewSelectedClick) {
+            statusElement = (
+                <PlainButton
```

Comment:
we should also probably give this and the one below `type="button"` because it defaults to `submit`

Reply:
err maybe only update this one but leave the other one